pages/data-migration/best-practices.mdx

```diff
@@ -66,20 +66,26 @@ href="/fundamentals/indexes"
 
 ### Use in-memory analytical storage mode
 
-Memgraph supports three storage modes: in-memory transactional, in-memory
-analytical and on-disk transactional. **To speed up import, use in-memory
-analytical storage mode**. On a larger scale, if you stick with the default
-in-memory transactional storage mode, you’ll experience slower import time and
-spend more resources during data import.  
+Memgraph supports three storage modes: `IN_MEMORY_TRANSACTIONAL`, `IN_MEMORY_ANALYTICAL` and `ON_DISK_TRANSACTIONAL`
+**To speed up import, use in-memory analytical storage mode**. 
 
 Data import is faster in in-memory analytical storage mode because `Delta`
 objects are disabled there, which ensures there is no memory overhead. The
 `Delta` objects track all changes, and that requires a lot of memory in case of
 frequent updates. Not having `Deltas` when importing data, speeds up the process
-significantly. The `Delta` objects are crucial to ensure durability and
-atomicity, so once you’re done with the data import, you might consider
+significantly. 
+
+The `Delta` objects are crucial to ensure atomicity, consistency, isolation and
+durability. If you are follwing instuctions, your import process does not need to depend on ACID properties, 
+and it is safe to use in-memory analytical storage mode, even in concurrent import scenarios.  
+
+Once you’re done with the data import process, if you have write enabled workloads you should consider
```

Comment:
Can you elaborate more on 'write enabled workloads'?

Reply:
If you do not have writes in your workload, there is no need to be in-memory transactional mode, since ACID mostly deals with write issues.  Tweaked a sentence a bit to be more clear. 

```suggestion
Once you’re done with the data import process, if you have write queries in your workloads,  consider
```

Reply:
Much better, thanks